build.gradle

```diff
@@ -275,22 +275,10 @@ task dockerBuild(type: Exec, dependsOn: shadowJar) {
 }
 
 task dockerBuildX(type: Exec, dependsOn: shadowJar) {
-    commandLine 'docker', 'buildx', 'build', '--load', '--platform', 'linux/arm64', '-t', dockerImageTag + '-arm64', project.rootDir
+    commandLine 'docker', 'buildx', 'build', '--push', '--platform', 'linux/arm64,linux/amd64,linux/arm/v7', '-t', dockerImageTag , project.rootDir
```

Comment:
dockerBuildX should uses the DockerfileX file instead of Dockerfile.

Reply:
I didn't use the `DockerfileX` file, because despite its existent, it wasn't being referred to by any other file in the repository, and was basically identical to the `Dockerfile` except for the `--platform` and the base image. Given BuildX is quite happy using the standard `Dockerfile`, what is the advantage of having an entirely separate file with just a couple of changed lines?